source/optional_features/delayed capture

```diff
@@ -0,0 +1,12 @@
+#Delayed capture 
+
+GOV.UK Pay supports delayed capture of payments.
+
+To use delayed capture, you can include `"delayed_capture": true` in the body of a request to create a payment. 
+
+The user experience matches the current payment flow. Once the user selects “Confirm” on the “Confirm your details” page, a request can be made by your service to capture the payment. 
+To do this, your service calls the POST /v1/payments/{paymentId}/capture endpoint. A 204 response indicates success; a 409 response indicates that the payment is not in an appropriate state to be captured. 
+
+You must make the capture request within 90 minutes of when the payment is created (regardless of how long the user takes to complete the payment flow) or it will expire.
```

Comment:
Would change to:

"Your service must send the delayed capture request within 90 minutes of when the payment is created, regardless of how long the user takes to complete the payment flow, or it will expire."

This avoids an embedded clause, which is in line with the style guide, and keeps the language consistent.